requirements.txt

```diff
@@ -1,12 +1,12 @@
 # uDMXController
-pyusb
-udmx-pyusb
+pyusb>=1.2.1,<2.0 
+udmx-pyusb>=2.0.0,<3.0
 
 # OpenDMXController
-pyftdi
+pyftdi>=0.54.0,<=0.55.0
 
 # Serial Controller
-pyserial
+pyserial>=3.5,<4.0
 
 # Web
-flask
\ No newline at end of file
+Flask>=2.3.3,<3.0
```

Comment:
Is there reason to pin this at 2.3.3 specifically, rather than just 2.3?

Reply:
Looking at the Flask Changelog, no not really so I changed to just 2.3. In other internal projects, I tend to use the latest working minor release version - just a habit I developed.